docs/ref/settings.txt

```diff
@@ -1836,6 +1836,20 @@ Here's a sample settings file::
         ('en', _('English')),
     ]
 
+.. setting:: LANGUAGES_BIDI
+
+``LANGUAGES_BIDI``
+------------------
+
+Default: A list of all languages in the :setting:`LANGUAGES` setting that are
```

Comment:
I think we should mention that by default it is a list of all languages codes from the **default** `LANGUAGES` setting that are written right-to-left, maybe with reference to the source code :thinking: .

Reply:
I think we can refer to the source code without listing the actual values so that this follows the same wording as for the `LANGUAGES` setting to keep things consistent.